packages/protocol/contracts/rewards/StakingRewards.sol

```diff
@@ -264,16 +284,49 @@ contract StakingRewards is ERC721PresetMinterPauserAutoIdUpgradeSafe, Reentrancy
     return maxRate.sub(maxRate.sub(minRate).mul(x.sub(intervalStart)).div(intervalEnd.sub(intervalStart)));
   }
 
-  function positionToLeveredAmount(StakedPosition storage position) internal view returns (uint256) {
-    return toLeveredAmount(position.amount, position.leverageMultiplier);
+  function positionToEffectiveLeveredAmount(StakedPosition storage position) internal view returns (uint256) {
+    uint256 effectiveAmountMultiplier = getEffectiveAmountMultiplier(position.positionType);
+    uint256 effectiveAmount = toEffectiveAmount(
+      position.amount,
+      position.baseTokenExchangeRate,
+      effectiveAmountMultiplier
+    );
+
+    return toLeveredAmount(effectiveAmount, position.leverageMultiplier);
+  }
+
+  function positionToEffectiveAmount(StakedPosition storage position) internal view returns (uint256) {
+    uint256 effectiveAmountMultiplier = getEffectiveAmountMultiplier(position.positionType);
+    return toEffectiveAmount(position.amount, position.baseTokenExchangeRate, effectiveAmountMultiplier);
+  }
+
+  function getEffectiveAmountMultiplier(StakedPositionType positionType) internal pure returns (uint256) {
+    if (positionType == StakedPositionType.Fidu) {
+      return MULTIPLIER_DECIMALS; // 100%
+    } else if (positionType == StakedPositionType.CurveLP) {
+      return MULTIPLIER_DECIMALS; // 100%
```

Comment:
I still think it's worth making this configurable via governance. Either by making an `onlyOwner` function here in StakingRewards, or by moving it to GoldfinchConfig.